airflow/contrib/utils/kubernetes_utils.py

```diff
@@ -37,6 +36,25 @@
 """
 
 
+def retryable_check_output(args, retry_count=3):
+    """
+    Reads the job description, retrying on failure
+    :param args: Arguments to pass to subprocess.check_output
+    :type args: List of string
+    :param retry_count: Number of times to retry (default=3)
+    :type retry_count: int
+    :return: string
+    """
+    try:
+        return subprocess.check_output(args=args)
```

Comment:
The codacy check wants you to change this line to read like:
`return subprocess.check_output(" ".join(args), shell=True)`